integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/TestEndpoint.java

```diff
@@ -1299,4 +1299,45 @@ public String testBug8254() {
 
         return "OK";
     }
+
+    @GET
+    @Path("9036")
+    @Transactional
+    public String testBug9036() {
+        Person.deleteAll();
+
+        Person emptyPerson = new Person();
+        emptyPerson.persist();
+
+        Person deadPerson = new Person();
+        deadPerson.name = "Stef";
+        deadPerson.status = Status.DECEASED;
```

Comment:
You need to be careful man... :laughing:

Reply:
It's a risky job…